src/widgets/AppWindow.js

```diff
@@ -2606,6 +2609,35 @@ export default class AppWindow extends createjs.Container {
     this._contentArea.accessible.addChild(tree);
   }
 
+  _mathTextCase() {
+    this._clearScreen();
+    
+    const options = {
+      src: formulaImg,
+      label: '(a+b)^{2}=a^{2}+2ab+b^{2}',
+      cjsScaleX: 1,
+      cjsScaleY: 1,
+    };
+    const mathText = new PlainTextMath(options);
+    mathText.x = 200;
+    mathText.y = 200;
+    this._contentArea.addChild(mathText);
+    this._contentArea.accessible.addChild(mathText);
+
+    const mathml= '<math xmlns ="http://www.w3.org/1998/Math/MathML"><mrow><msup><mi>a</mi><mn>2</mn></msup><mo>+</mo><msup><mi>b</mi><mn>2</mn></msup><mo> = </mo><msup><mi>c</mi><mn>2</mn></msup> </mrow></math>';
+    const text = new createjs.Text('a2+b2=c2','bold 20px Arial', '#000');
```

Comment:
Shouldn't the createjs.Text instance be created by the class in MathML.js (either through composition or inheritance) and an instance of that class used here?

Reply:
yes, as MathML is not needed text can be created using PlainTextMath.